src/lib/Zikula/Core/Helper/ExtensionVar.php

```diff
@@ -0,0 +1,283 @@
+<?php
+/**
+ * Copyright 2015 Zikula Foundation
+ *
+ * This work is contributed to the Zikula Foundation under one or more
+ * Contributor Agreements and licensed to You under the following license:
+ *
+ * @license GNU/LGPLv3 (or at your option, any later version).
+ * @package Zikula
+ *
+ * Please see the NOTICE file distributed with this source code for further
+ * information regarding copyright and licensing.
+ */
+
+namespace Zikula\Core\Helper;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Zikula\Core\Doctrine\Entity\ExtensionVarEntity;
+
+/**
+ * Class ExtensionVar
+ * @package Zikula\Core\Helper
+ *
+ * This class manages the storage and retrieval of extension variables and is the intended replacement
+ * for ModUtil::* methods (getVar, setVar, etc) as well as similar functionality in System:: and ThemeUtil::
+ */
+class ExtensionVar
+{
+    const CONFIG_MODULE = 'ZConfig';
+
+    /**
+     * @var EntityManagerInterface
+     */
+    private $entityManager;
+    /**
+     * @var KernelInterface
+     */
+    private $kernel;
+
+    /**
+     * @var \ArrayObject
+     * @todo consider \Symfony\Component\DependencyInjection\ParameterBag\ParameterBag
+     * @todo consider \Symfony\Component\HttpFoundation\ParameterBag
```

Comment:
Why is this an ArrayObject in the first place? I don't see why a simple array won't work.